azurerm/resource_arm_application_gateway_test.cer

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
that's fine - as discussed in DM we can instead use a hard-coded certificate for now, and perhaps switch this out at a later date if the `tls` provider supports this functionality